### DISCUSSION TITLE
Issue with wildcards and Combinatorial generation

### DISCUSSION THREAD

**Discussion by manzomium** [adieyal/sd-dynamic-prompts#97] [Q&A]
Here I am again. I've played a bit with Combinatiorial generation + wild cards as is "portrait of {man|woman} during {day|night} by __artist__" and the extension seems to just pick the first name of the wildcard list, while it woks fine for the combination. Is that something to be expected? meaning: you know about it and would work on a solution? ;-)

**Comment by adieyal** [comment]
Are you saying that when you use combinatorial generation, only the first wildcard is chosen from the file? Could you describe what you're doing step by step so that I can try figure out the issue?

**Reply by manzomium** [reply]
I guess I'm having the same issue as reported here: https://github.com/adieyal/sd-dynamic-prompts/discussions/104
Indeed, it only happens when generating batch img2img

**Reply by adieyal** [reply]
Have you tried the suggested solution? https://github.com/adieyal/sd-dynamic-prompts/issues/107

**Reply by adieyal** [reply]
If #107 works for you, could you please mark this question as answered? Otherwise it is hard for me to keep track of unanswered questions.

**Reply by manzomium** [reply]
Done!

**Reply by manzomium** [reply]
Hey, I'm back to this issue again, hope you can help...

It seemed to work when I checked "unlink seed from prompt", and it generated woman from __Arab-nationalities__ using a combination of places {kasbah|mosque|oasis} and of different clothes {burqa|chador|jihab}. It worked just fine.
Now it is not: I'm trying to generate woman of different age stages {30|40|50} of __nationalities__ in {Milan|Paris|Seattle}. It gives only three wildcard variations while the combination with age and place works just fine. Maybe is it something I need to do with the number of batches?
